app/lua53/lauxlib.c

```diff
@@ -1153,6 +1154,8 @@ static int errhandler_aux (lua_State *L) {
 ** to convert into an error message which it handles in a separate posted task.
 */
 static int errhandler (lua_State *L) {
+  if (!strncmp(lua_tostring(L, -1), "!LFS reload!", sizeof("!LFS reload!") - 1))
```

Comment:
It's not really worth using `strncmp` when one of the parameters is a constant; go ahead and use `strcmp` without the repetition.  The compiler's going to inline it anyway, one imagines.

Reply:
Yup `memcmp` is better.  We can't use `strcmp` because the various reload messages include a qualification.

Reply:
"include a qualification"?